docker/kafka-monitor-docker-entry.sh

```diff
@@ -15,7 +15,13 @@
 
 set -x
 
+# SIGTERM-handler
+trap 'pkill java; exit 129' SIGINT
```

Comment:
According to https://en.wikipedia.org/wiki/Unix_signal, the port number for SIGINT is 2. Should this be 130 since 128 + 2 = 130?